### PR TITLE
update .gitignore and .env.template

### PR DESCRIPTION
**Pull request by zberwaldt**

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
 # Discord application
+#
+# This is the token for the discord bot.
 DISCORD_BOT_TOKEN=
+# This is the public key for the discord bot.
 DISCORD_BOT_PUBLIC_KEY=
+# This is the application id for the discord bot.
 DISCORD_BOT_APPLICATION_ID=
+# This is the id of the owner of the discord bot. For local development you can use your own id.
 DISCORD_OWNER_ID=
```
Comment (0xtiki) [inline .env.template]: Not needed

File: .env.template [hunk 2]
```diff
@@ -1,17 +1,35 @@
...
+# This is the activity the bot will display in discord.
+DISCORD_BOT_ACTIVITY=
```
Comment (0xtiki) [inline .env.template:17]: Maye we can add a default for this to ready.ts? It's not really that important so it should't be mandatory to set it. E.g. "Voting with Tokens"

File: .env.template [hunk 3]
```diff
@@ -1,17 +1,35 @@
...
-MONGODB_PASS=
-MONGODB_CLUSTER=
-
-# Logger
-LOGDNA_APP_NAME=
-LOGDNA_DEFAULT_LEVEL=
-LOGDNA_TOKEN=
+# Governator API
+#
+# For local development base path will be your local governator api url
```
Comment (0xtiki) [inline .env.template]: e.g. `http://localhost:4500`

File: .env.template [hunk 4]
```diff
@@ -1,17 +1,35 @@
...
+GOVERNATOR_API_BASE_PATH=
+# For local development you can use the governator api key from the governator api
+GOVERNATOR_API_KEY=
+# this is used to build the url for the governator api
```
Comment (0xtiki) [inline .env.template]: e.g. `api`

File: .env.template [hunk 5]
```diff
@@ -1,17 +1,35 @@
...
+GOVERNATOR_API_PREFIX=
+# this is used to build the url for the governator api
```
Comment (0xtiki) [inline .env.template:27]: should be set to `sse` unless you customized the endpoint on the api

File: .env.template [hunk 6]
```diff
@@ -1,17 +1,35 @@
...
+GOVERNATOR_API_ENDPOINT_SSE=
+# This if for the governator api logo.
```
Comment (0xtiki) [inline .env.template:29]: maybe we can set a default here too? E.g. `https://www.governator.xyz/images/gov-bot.jpeg`

File: .env.template [hunk 7]
```diff
@@ -1,17 +1,35 @@
...
+GOVERNATOR_LOGO_URL=
 
+# Logging
```
Comment (0xtiki) [inline .env.template]: optional, defaults to `info`, possible values are `error`, `warn`, `info`, `debug`, `data`

File: .gitignore [hunk 8]
```diff
@@ -118,4 +118,7 @@ build
 # jest-mongodb output
 globalConfig.json
 
-https://discord.com/api/oauth2/authorize?client_id=901799442502516766&permissions=4027055222&scope=bot%20&applications.commands
\ No newline at end of file
+https://discord.com/api/oauth2/authorize?client_id=901799442502516766&permissions=4027055222&scope=bot%20&applications.commands
```
Comment (0xtiki) [inline .gitignore]: not sure how that ended up in there, but we can remove that :)

**Review by 0xtiki — COMMENTED** [review]
Thank you @zberwaldt for updating this. I left some suggestions that should help clarify a few more things.

File: .gitignore (hunk 8 above)
Comment (zberwaldt) [reply .gitignore]: done

File: .env.template (hunk 3 above)
Comment (zberwaldt) [reply .env.template]: This should be resolved

File: .env.template (hunk 5 above)
Comment (zberwaldt) [reply .env.template:27]: This should be resolved

File: .env.template (hunk 6 above)
Comment (zberwaldt) [reply .env.template:29]: This should also be resolved

File: src/app/service/poll/CreatePoll.ts [hunk 9]
```diff
@@ -161,7 +161,7 @@ const pollEmbed = async (pollEmbedParams: PollEmbedParams): Promise<EmbedBuilder
             iconURL: (pollEmbedParams.author as Discord.User).avatarURL() as string,
         })
         .setFooter({ text: pollEmbedParams.poll._id as string })
-        // .setThumbnail(process.env.GOVERNATOR_LOGO_URL?.toString() as string)
+        .setThumbnail((process.env.GOVERNATOR_LOGO_URL?.toString() ?? 'https://www.governator.xyz/images/gov-bot.jpeg') as string)
```
Comment (0xtiki) [inline src/app/service/poll/CreatePoll.ts:164]: Actually this was off on purpose, let's keep that commented out. If that is the only place GOVERNATOR_LOGO_URL is used, we can just remove it from the env template.

**Review by 0xtiki — COMMENTED** [review]
Found one last issue with the thumbnail, it should be off as it messes up the layout of the embed. Otherwise looks good to me.

**Pull request closed without merging**